src/error.rs

```diff
@@ -10,6 +11,7 @@ pub enum Error {
     RustcNotFound,
     Io(IoError),
     GlobPatternError(&'static str),
+    Toml(PathBuf, TomlError),
```

Comment:
@dvc94ch Before merging this, shall I do the same for at least `IoError`? It's a bit annoying to see "No such file or directory" without knowing what the path was.

Reply:
Sounds good. Wish io errors were more descriptive by default.

Reply:
Done! Would you mind reviewing+merging this and #13? I'll update #12 after that with some fixes.